index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <header>
+        <div>
+            <img id="header-img1" src="media/image_geometry_1.png">
+            <img id="header-img2" src="media/image_geometry_2.png">
+        </div>
+        <nav>
+            <a>
+                <img id="menu" src="media/ic_menu.png">
+            </a>
+        </nav>
+    </header>
+    <headline class="headline">
+        Glenda Diaz
```

Comment:
- [OPTIONAL] This is the main title of the page and it should be wrapped in an `h1` tag. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.